pkgs/tools/networking/qodem/default.nix

```
@@ -0,0 +1,28 @@
+{ lib, stdenv, fetchFromGitHub, autoconf, automake, ncurses, SDL, gpm, miniupnpc }:
+stdenv.mkDerivation rec {
```

[nit] by convention there's usually a newline after the arguments list

```suggestion

stdenv.mkDerivation rec {
```

---

Done!